### PR TITLE
Add MeeseeksDev / Lumberbot (App) to GitHub page

### PR DESCRIPTION
**Pull request by matthewfeickert**
* c.f. https://github.com/scientific-python/MeeseeksDev

### CODE REVIEW

File: content/actions/index.md [hunk 1]
```diff
@@ -55,4 +55,11 @@ body = '''
 Add a GitHub status link directly to a CircleCI artifact.
 '''
 
+[[item]]
+type = 'card'
+header = '[MeeseeksDev / Lumberbot (App)](https://github.com/scientific-python/MeeseeksDev)'
```
Comment (matthewfeickert) [inline content/actions/index.md:60]: @Carreau has moved the bot to the Scientific Python org, but we're going to rename it later once we have validated everything works. We can do a follow up PR once that has been done.

**Comment by matthewfeickert** [comment]
Sorry, I forgot that I could just hit @scientific-python/tools-team for review.

File: content/actions/index.md (hunk 1 above)
Comment (matthewfeickert) [reply content/actions/index.md:60]: c.f. https://github.com/scientific-python/MeeseeksDev/issues/99#issuecomment-2151144959

**Review by stefanv — APPROVED** [review]

**Review by j-bowhay — APPROVED** [review]
Thanks @matthewfeickert

**Review by bsipocz — APPROVED** [review]
Looks good for now, but I would suggest a follow up to separate the actions and apps into either separate sections in the same page, or into separate pages. 

I would prefer the separate sections as we only have one bot.

**Pull request merged**